cmd/dfget/app/root.go

```diff
@@ -234,8 +242,8 @@ func initFlags() {
 			"\nin this way, different but actually the same URLs can reuse the same downloading task")
 	flagSet.StringSliceVar(&cfg.Header, "header", nil,
 		"http header, eg: --header='Accept: *' --header='Host: abc'")
-	flagSet.StringSliceVarP(&cfg.Nodes, "node", "n", nil,
-		"specify the addresses(host:port) of supernodes")
+	flagSet.StringSliceVarP(&cfg.Supernodes, "node", "n", nil,
+		"specify the addresses(host:port/weight) of supernodes where the host is necessary, the port(default: 8002) and the weight(default:1) are optional.")
```

Comment:
I think it's necessary to tell the user weight must be an integer.  Maybe we can also give some advice on how to set the weight, e.g. use multiples of 10